DS_Assignment_6/Employee/Employee.java

```diff
@@ -0,0 +1,43 @@
+package Employee;
+
+public class Employee implements Comparable<Employee> {
+
+	private int emp_ID;
+	private String name;
+	private String address;
+
+	public Employee(int emp_ID, String name, String address) {
+		this.emp_ID = emp_ID;
+		this.name = name;
+		this.address = address;
+	}
+
+	public int getEmpID() {

```

Comment:
It seems you only created getter method for the class fields. where are the setter methods?